.travis.yml

```diff
@@ -0,0 +1,21 @@
+ branches:
+  only:
+    - module-7
```

Comment:
Don't forget to remove this when you're ready to merge; we want these tests to run on every branch going forward

Reply:
Done